webpack/webpack.config.js

```diff
@@ -168,6 +169,33 @@ module.exports = {
           }
         ]
       },
+      {
+        test: /\.css$/,
```

Comment:
## 🔴 

same as above. why do we need a fully fledged css-config just for graphiQL?

why do we need a `postcss`-config to include what looks like regular css?

Reply:
Similar reason as above